rpc/backend/src/test/scala/io/udash/rpc/ClientRPCTest.scala

```diff
@@ -1,55 +1,66 @@
 package io.udash.rpc
 
 import com.github.ghik.silencer.silent
-import io.udash.rpc.internals.{BroadcastManager}
-import io.udash.testing.{RunNowExecutionContext, UdashBackendTest}
-import org.atmosphere.cpr.{Broadcaster, BroadcasterFactory, MetaBroadcaster}
+import io.udash.rpc.internals.BroadcastManager
+import io.udash.testing.UdashRpcBackendTest
+import org.atmosphere.cpr._
 
 import scala.concurrent.ExecutionContext
 
-class ClientRPCTest extends UdashBackendTest {
-  implicit val ec = RunNowExecutionContext
+class ClientRPCTest extends UdashRpcBackendTest {
+  implicit val ec = scala.concurrent.ExecutionContext.Implicits.global
 
   def tests(createClientRpc: (ClientRPCTarget) => ClientRPC[TestClientRPC]) = {
     "gain access to RPC methods of concrete client" in {
       BroadcastManager.synchronized {
-        val broadcasterFactory = mock[BroadcasterFactory]
-        val metaBroadcaster = mock[MetaBroadcaster]
-        val broadcaster = mock[Broadcaster]
-
-        (broadcasterFactory.lookup[Broadcaster](_: Any, _: Boolean)).expects("/client/id123", *).twice().returns(broadcaster)
-        (broadcaster.broadcast(_: Any)).expects(*).twice()
+        BroadcastManager.init(null, null)
```

Comment:
you call `init(null, null) on the beginning of each test to re-setup `BroadcastManager`.
Personally I don't like to use an `object` when something has mutable state. 

Is it possible to make class out of `BroadcastManager`?
Apart from tests it's used only in two places (AtmosphereService and ClientRPC), but I don't know how they are used, so I don't know whether is it doable.

If manager where a class then you could create new instance for each test instead of re-initializing (which even logs warnings)